src/apb/engine.py

```diff
@@ -707,6 +770,35 @@ def print_list(services):
         print(template.format(**service))
 
 
+def build_apb(project, dockerfile=None, registry=None, org=None, tag=None):
+    if dockerfile is None:
+        dockerfile = "Dockerfile"
+    spec = get_spec(project)
+    if 'version' not in spec:
+        print("APB spec does not have a listed version. Please update apb.yml")
+        exit(1)
+
+    name = "{}{}{}".format(
+        registry + "/" if registry is not None else "",
```

Comment:
What happens if I set `--tag` and `--registry`?

Reply:
Looking at how `apb build` works. I took `tag` to be the name of the apb (since if it is not defined we take the name from the apb.yml). Not sure how we want to do this.